src/styles/main.scss

```diff
@@ -0,0 +1,50 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  counter-reset: day-counter 0;
+  font-family: Arial, sans-serif;
+}
+
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(7, $day-size);
```

Comment:
Ensure that the variable `$day-size` is defined in your SCSS files. This variable is used to set the size of each day in the calendar grid.